File: C language/Assignment-6/LCM of two numbers.c
#include<stdio.h>
#include<conio.h>
int main()
{
    int n1,n2,i=1,lcm;
    printf("Enter the first number:");
    scanf("%d",&n1);
    printf("Enter the second number:");
    scanf("%d",&n2);

   while(i<=n1*n2)
    {
       if(i%n1==0 && i%n2==0 )
       {
          lcm=i;
          break;
       }
        i++;
    }
    printf("The LCM of %d and %d is %d",n1,n2,lcm);
    return 0;
}

File: Python MySirG/Assignment-12/LCM of two numbers.py
n1=int(input("Enter the first number:"))
n2=int(input("Enter the second number:"))
if n1>n2:
    u=n1
    l=n2
else:
    u=n2
    l=n1
LCM=1
i=1
while LCM%l!=0:
    LCM=u*i
    i+=1
print("LCM=%d"%LCM)
